frontend/src/components/UserSettings/UserGmsLocationCapturing.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <BButton @click="isModalOpen = !isModalOpen">{{ $t('settings.GMS.location.button') }}</BButton>
+  <BModal :model-value="isModalOpen" fullscreen @update:modelValue="isModalOpen = !isModalOpen">
+    <template #title>
+      <h3>{{ $t('settings.GMS.map.headline') }}</h3>
+    </template>
+    <template #default>
+      <BContainer class="bg-white appBoxShadow gradido-border-radius p-4 mt--3">
+        <user-location-map
+          v-if="isModalOpen"
+          :user-marker-coords="userLocation"
+          :community-marker-coords="communityLocation"
+          @update:userPosition="updateUserLocation"
+        />
+      </BContainer>
+    </template>
+    <template #footer>
+      <BButton variant="gradido" @click="saveUserLocation">
+        {{ $t('settings.GMS.location.saveLocation') }}
+      </BButton>
+    </template>
+  </BModal>
+</template>
+
+<script setup>
+import { ref, computed } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { useMutation, useQuery } from '@vue/apollo-composable'
+import 'leaflet/dist/leaflet.css'
+import { updateUserInfos } from '@/graphql/mutations'
+import { useAppToast } from '@/composables/useToast'
+import UserLocationMap from '@/components/UserSettings/UserLocationMap'
+import { BButton, BModal } from 'bootstrap-vue-next'
+import { userLocationQuery } from '@/graphql/queries'
+import CONFIG from '@/config'
+
+const { t } = useI18n()
+const { mutate: updateUserInfo } = useMutation(updateUserInfos)
+const { onResult, onError } = useQuery(userLocationQuery, {}, { fetchPolicy: 'network-only' })
+const { toastSuccess, toastError } = useAppToast()
+
+const capturedLocation = ref(null)
+const isModalOpen = ref(false)
+const userLocation = ref({ lat: 0, lng: 0 })
+const communityLocation = ref({ lat: 0, lng: 0 })
+
+const emit = defineEmits(['close'])
+
+onResult(({ data }) => {
+  communityLocation.value.lng = data.userLocation.longitude
+  communityLocation.value.lat = data.userLocation.latitude
+
+  userLocation.value.lng = data?.userLocation?.longitude ?? communityLocation.value.lng
+  userLocation.value.lat = data?.userLocation?.latitude ?? communityLocation.value.lat
+})
+
+onError((err) => {
+  userLocation.value = defaultLocation.value
+  communityLocation.value = defaultLocation.value
+  toastError(err.message)
+})
+
+const defaultLocation = computed(() => {
+  const defaultCommunityCoords = CONFIG.COMMUNITY_LOCATION.split(',')
+  return {
+    lat: parseFloat(defaultCommunityCoords[0]),
+    lng: parseFloat(defaultCommunityCoords[1]),
+  }
```

Comment:
Is this the correct index usage for the right coordinates?
[0] = latitude
[1] = longitude

Because in "frontend/src/config/index" the property
CONFIG.COMMUNITY_LOCATION is defined with the default settings '49.280377, 9.690151'

Reply:
Wenn ich die Koordinaten bei google maps eingebe kommt Künzlsau dabei heraus und google formatiert mir das so:
49°16'49.4"N 9°41'24.5"E
Wenn ich das chatgpt gebe kommt das dabei heraus:
49°16'49.4"N ist die Breitengrad (Latitude).
9°41'24.5"E ist der Längengrad (Longitude).